pkg/fileservice/local_etl_fs.go

```diff
@@ -475,13 +475,25 @@ type _LocalETLFSMutator struct {
 }
 
 func (l *_LocalETLFSMutator) Mutate(ctx context.Context, entries ...IOEntry) error {
+	return l.mutate(ctx, 0, entries...)
+}
+
+func (l *_LocalETLFSMutator) Append(ctx context.Context, entries ...IOEntry) error {
```

Comment:
What is the reason using a leading _?   _ will make this public or private in go?   Why should we do this ...

Reply:
I wanted to make it private. But now I think it's OK to be public.